ci/travis/determine_tests_to_run.py

```diff
@@ -207,6 +207,9 @@ def get_commit_range():
             elif changed_file.endswith("build-docker-images.py"):
                 RAY_CI_DOCKER_AFFECTED = 1
                 RAY_CI_LINUX_WHEELS_AFFECTED = 1
+            elif changed_file.startswith("python/ray/tests/kuberay"):
```

Comment:
wait is this right? shouldn't we rerun this whenever autoscaler changes? not when the tests change?

Reply:
Sorry, I meant to drop this.

Reply:
Oh, I did drop this. This is was in the middle of pipeline drafting.

The tests will run whenever "RAY_CI_WHEELS_AFFECTED", which actually includes test changes and autoscaler changes.